mockserver-core/src/test/java/org/mockserver/codec/BodyDecoderEncoderTest.java

```diff
@@ -38,7 +38,7 @@ public void shouldSerialiseBodyToByteBufWithNoContentType() {
     }
 
     @Test
-    public void shouldSerialiseBodyToByteBufWithIvalidContentType() {
+    public void shouldSerialiseBodyToByteBufWithInvalidContentType() {
```

Comment:
A lot of these are halfway between things...

I can't figure out if this was the right choice based on reading the implementation. `image/png` is a valid content type....